gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/GoogleCloudStorageGrpcIntegrationTest.java

```diff
@@ -179,6 +179,19 @@ public void testOpenWithMetricsEnabled() throws IOException {
     assertObjectContent(rawStorage, objectToCreate, objectBytes);
   }
 
+  @Test
+  public void testOpenWithTracingLogEnabled() throws IOException {
```

Comment:
Let's add an integration test for the JSON path as well. Ensuring no regressions

Reply:
bump

Reply:
I will add this with the next change when we actually consume the tracer.